sdk/search/Azure.Search.Documents/tests/FieldBuilderTests.cs

```diff
@@ -416,11 +416,10 @@ public void FieldBuilderFailsWithHelpfulErrorMessageOnUnsupportedPropertyTypes(T
             string expectedErrorMessage =
                 $"Property '{invalidPropertyName}' is of type '{modelType.GetProperty(invalidPropertyName).PropertyType}', " +
                 "which does not map to an Azure Search data type. Please use a supported data type or mark the property with " +
-                "[FieldBuilderIgnore] and define the field by creating a SearchField object. See https://aka.ms/azsdk/net/search/fieldbuilder for more information." +
-                $"{Environment.NewLine}Parameter name: {nameof(modelType)}";
```

Comment:
Why did you change this? I used the same format that ArgumentExceptions do by design.

Reply:
ArgumentExceptions message format has changed in 3.1 so all assets that were using an exact format have to be changed.